Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM node:20.1.0-buster-slim AS node-image
-FROM python:3.11.3-slim-buster
+FROM node:20.11-bookworm-slim AS node-image
+FROM python:3.11.7-slim-bookworm
 
 # Requirements for building packages
 RUN apt-get update \
   && apt-get install -y --no-install-recommends \
         bzip2 ccache f2c g++ gfortran git make \
         patch pkg-config swig unzip wget xz-utils \
         autoconf autotools-dev automake texinfo dejagnu \
-        build-essential prelink autoconf libtool libltdl-dev \
```

Comment:
@ryanking13 @rth does anyone know what prelink does for us or why it went away in bookworm?

Reply:
You added it in https://github.com/pyodide/pyodide/pull/1656 so I don't know exactly why we need it :) 

Looking at https://lwn.net/Articles/341244/ and https://forums.gentoo.org/viewtopic-t-1074192-start-0.html sounds like it was some debatable optimization, so maybe they just removed it.

Reply:
It seems likely that I copied it out of docker files for building Python since I do find it there. Thanks for investigating @rth!